cnf-tests/.konflux/README.md

```diff
@@ -31,10 +31,35 @@ When an image version is out-of-maintenance (OOM) some versions has what's calle
 As any other RPM repo, also EUS repos need to be enabled in the activation key. Once enabled, the lockfile will be generated with additional EUS packages. The version of the base images should anyhow align with those used for OCP for the same branch.   
 
 **Important**: 
-* When starting the container in which you will be generating the lockfile in, use a production image in order to get the GA RPM repos and not beta one. So use `registry.redhat.io/ubi9/ubi-minimal:9.4` and not `registry-proxy.engineering.redhat.com/rh-osbs/ubi9/ubi-minimal:9.4`.
+* When starting the container in which you will be generating the lockfile in, use a production image in order to get the GA RPM repos and not beta one. So use `registry.access.redhat.com/ubi9/ubi-minimal:9.4` and not `registry-proxy.engineering.redhat.com/rh-osbs/ubi9/ubi-minimal:9.4`.
 * Please make sure that the repos that you used to pull the RPMs from are found under the activation key that is associated to the konflux public instance by:
 <steps on how to confirm this will be detailed later once we have a team activation key> 
 
 **RPM automatic updates**
 Konflux uses a mechanism to automatically file PRs to update RPM versions and generate the updated lockfile, and is called Mintmaker. However, this is supported only for repos that have the input file saved in the root of the project repository, which is not an ideal place for cnf-features-deploy considering it produces multiple images (cnf-tests, ztp,..).
-The support for multiple data sources (input files) is in progress at https://issues.redhat.com/browse/CWFHEALTH-3922.
\ No newline at end of file
+The support for multiple data sources (input files) is in progress at https://issues.redhat.com/browse/CWFHEALTH-3922.
+
+**Manual RPM lock regeneration using Makefile**
+To manually regenerate the rpm lock configuration for cnf-tests, use the following Makefile targets from the `cnf-tests/` directory.
+
+1. **Update rpm lock file for runtime:**
+   ```bash
+   make konflux-update-rpm-lock-runtime
+   ```
+   This target will:
+   - Sync git submodules
```

Comment:
is this a hard requirement for updating the lockfile? 
I would prefer small self-contained updates in the rpmlockfile update PR that only addresses the bump of the rpm dependencies